apps/browser/src/autofill/background/notification.background.ts

```diff
@@ -434,11 +445,13 @@ export default class NotificationBackground {
 
   private async pushUnlockVaultToQueue(loginDomain: string, tab: chrome.tabs.Tab) {
     this.removeTabFromNotificationQueue(tab);
+    const launchTimestamp = new Date().getTime();
     const message: AddUnlockVaultQueueMessage = {
       type: NotificationQueueMessageType.UnlockVault,
       domain: loginDomain,
       tab: tab,
-      expires: new Date(new Date().getTime() + 0.5 * 60000), // 30 seconds
+      launchTimestamp,
+      expires: new Date(launchTimestamp + 0.5 * 60000), // 30 seconds
```

Comment:
non-blocking nit; I just prefer to not rely on order of operations (same on line 481). up to you 🤷 

```suggestion
      expires: new Date(launchTimestamp + 30000), // 30 seconds
```